backend/controllers/ingredient_controller.js

```diff
@@ -2,29 +2,26 @@ const db = require("../models");
 const ingredients = db.ingredient;
 
 // retrieves ingredient by primary key
-exports.get = async(req,res) => {
+exports.get = async(req,_res) => {
```

Comment:
If we are going to use _req,_res in some methods, we should use them in all methods that are similar

Reply:
They are meant to signify unused parameters and as such, it would not make sense for parameters that are being used.